gratipay/cli/sync_npm.py

```diff
@@ -3,40 +3,35 @@
 """
 from __future__ import absolute_import, division, print_function, unicode_literals
 
-import sys
-import argparse
-
-from gratipay import wireup
-from gratipay.sync_npm import serialize, upsert
-
-
-def parse_args(argv):
-    p = argparse.ArgumentParser()
-    p.add_argument('command', choices=['serialize', 'upsert'])
-    p.add_argument('path', help='the path to the input file', nargs='?', default='/dev/stdin')
-    return p.parse_args(argv)
+import time
 
+from aspen import log
 
-subcommands = { 'serialize': serialize.main
-              , 'upsert': upsert.main
-               }
+from gratipay import wireup
+from gratipay.sync_npm import consume_change_stream, get_last_seq, production_change_stream
+from gratipay.utils import sentry
 
 
-def main(argv=sys.argv):
+def main():
     """This function is installed via an entrypoint in ``setup.py`` as
     ``sync-npm``.
 
     Usage::
 
-      sync-npm {serialize,upsert} {<filepath>}
-
-    ``<filepath>`` defaults to stdin.
-
-    .. note:: Sphinx is expanding ``sys.argv`` in the parameter list. Sorry. :-/
+      sync-npm
 
     """
     env = wireup.env()
-    args = parse_args(argv[1:])
     db = wireup.db(env)
-
-    subcommands[args.command](env, args, db)
+    while 1:
+        with sentry.teller(env):
+            consume_change_stream(production_change_stream, db)
+        try:
+            last_seq = get_last_seq(db)
```

Comment:
Hmm, if we're calling `get_last_seq` here anyway - might make sense to simplify the function definition of `consume_change_stream` to accept the stream directly, and not a function that has to be called with `seq` to return the stream?